docker-compose-test.yaml

```diff
@@ -30,7 +30,12 @@ services:
 
   test:
     image: python:3.8-alpine3.18
```

Comment:
is there a good reason to use different image in "lint" (`python:3.8.18-slim-bullseye`) and in test (`python:3.8-alpine3.18`) jobs?